main.py

```diff
@@ -1,19 +1,48 @@
 import os
+import json
 from typing import Any
 
 import requests
 from flask import Flask, render_template
 
+from google.oauth2 import service_account
+from google.auth.transport.requests import Request
+
 JsonDict = dict[str, Any]
 
 NYCOMP_PAT = os.environ.get("GH_PAT")
+# Typo is intentional; when key is renamed, will change this too
+SERVICE_ACCOUNT_JSON = json.loads(
+    os.environ.get('GOOGLE_SEVICE_ACCOUNT')
+)
```

Comment:
```suggestion
SERVICE_ACCOUNT_JSON = json.loads(
    os.environ['GOOGLE_SEVICE_ACCOUNT']
)
```
Use dict[] syntax since `None` is not an expected input to `json.loads()` (and we want to be informed via a raised error if the env var is missing)